.circleci/config.yml

```diff
@@ -334,7 +334,7 @@ workflows:
       - test_solidus:
           context: slack-secrets
           name: &name "test-rails-<<matrix.rails>>-ruby-<<matrix.ruby>>-<<matrix.database>>-<<#matrix.paperclip>>paperclip<</matrix.paperclip>><<^matrix.paperclip>>activestorage<</matrix.paperclip>>"
-          matrix: { parameters: { rails: ['7.0'], ruby: ['3.0', '3.1'], database: ['mysql', 'sqlite', 'postgres'], paperclip: [true, false] } }
+          matrix: { parameters: { rails: ['7.0'], ruby: ['3.1', '3.2'], database: ['mysql', 'sqlite', 'postgres'], paperclip: [true, false] } }
```

Comment:
Thoughts on keeping 3.0 + rails 7 around?

Reply:
I think it still would be better, and ensure we have a ruby version tested for adjacent rails versions, e.g. having ruby 3.0 be the shared version between rails 7 and 6.1, but it's just a nice to have.

It's fine if we merge and then add it later 👍👍👍

Reply:
Ok, that makes a lot of sense. I think we might accomplish the same result by testing Ruby 3.1 compatibility with Rails 6.1.

Reply:
Done!